src/app/project/board/dataset-metadata.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { IId, Organization, Person } from '@dasch-swiss/dsp-js';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DatasetMetadataService {
```

Comment:
I would just name it `MetadataService`, or `MetadataUtilService` where all helpers method could be stored.

Also this repository has using 4 spaces indentions.

Reply:
done in [d8b4993](https://github.com/dasch-swiss/dsp-app/pull/400/commits/d8b4993b6648e3fe469ecdf0016843e8d0b8d78c).